wallet_anchor.go

```diff
@@ -231,6 +233,23 @@ func (l *LndRpcWalletAnchor) MinRelayFee(
 	return l.lnd.WalletKit.MinRelayFee(ctx)
 }
 
+// DeriveNextKey derives the next key in the taproot assets key family.
+func (l *LndRpcWalletAnchor) DeriveNextKey(
```

Comment:
What about `key_ring.go`?

Reply:
Ah, I see. Perhaps in that case we should split the `supplycommit.Wallet` into `supplycommit.Wallet` and `supplycommit.KeyRing` to avoid this duplication? Or merge the `NewLndRpcKeyRing` and `LndRpcWalletAnchor` wallet types into a single type that implements both?